README.md

```diff
@@ -62,85 +62,39 @@ Then start the `xremap` daemon by running:
 sudo xremap config.yml
 ```
 
-(You will need to leave it running for your mappings to take effect.)
+To use it without sudo (recommended) run the following commands:
```

Comment:
It's a lot of work to make it work, so I actually don't recommend it. It's still often necessary to make `application`-specific remappings work in many platforms. Let's remove "(recommended)" and indicate that it's needed for `application` filters.